english/nfl-chiefs/nfl-chiefs.txt

```diff
@@ -0,0 +1 @@
+Hello, I like the chiefs. Do you like the Chiefs? If you don’t, you must be a robot.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.